public/app/features/trails/TrailStore/TrailStore.ts

```diff
@@ -23,9 +24,14 @@ export interface SerializedTrail {
   createdAt?: number;
 }
 
+export interface SerializedBookmarkState {
```

Comment:
```suggestion
export interface DataTrailBookmark {
```

Not sure the sure the Seralized and Sate prefix/suffix adds much